src/components/images/song-image.tsx

```diff
@@ -0,0 +1,27 @@
+import { MusicalNoteIcon } from "@heroicons/react/24/solid"
+import React from "react"
+
+export interface SongImageProps {
+  url: string | null
+}
+
+const SongImage = ({ url }: SongImageProps) => {
+  return <>
+    {url
+      ? <img
```

Comment:
Kan je hier niet de Image tag van next gebruiken?

Reply:
no because the images need to be rendered client side